cluster-scope/overlays/nerc-ocp-infra/persistentvolumeclaims/openshift-storage-db-noobaa-db-pg-0.yaml

```diff
@@ -0,0 +1,22 @@
+kind: PersistentVolumeClaim
+apiVersion: v1
+metadata:
+  annotations:
+    pv.kubernetes.io/bind-completed: 'yes'
+    pv.kubernetes.io/bound-by-controller: 'yes'
+    volume.beta.kubernetes.io/storage-provisioner: openshift-storage.rbd.csi.ceph.com
+    volume.kubernetes.io/storage-provisioner: openshift-storage.rbd.csi.ceph.com
+  resourceVersion: '657199373'
```

Comment:
This should be removed.

```suggestion
```

Reply:
Thanks @larsks for helping me clean this up, I updated my commit to remove the resourceVersion.